src/openrct2/network/NetworkAction.h

```diff
@@ -15,11 +15,11 @@
 #include <string>
 #include <vector>
 
-enum MISC_COMMAND
+enum class MiscCommand : int32_t
```

Comment:
This should be combined with the `GAME_COMMAND` enum as that is what they are part of.

Reply:
Yep, sure thing. I'm working on merging these now.

Reply:
Actually that is not correct, they are only used in the network protocol and have no relation to game commands/game actions really. This should be reverted back to the way it was.